ensime_shared/spec/features/server_bootstrap_steps.py

```diff
@@ -0,0 +1,66 @@
+import os
+import shutil
+import subprocess
+from os import path
+
+import psutil
+from lettuce import step, world
+
+from ensime_shared.launcher import EnsimeLauncher
+
+
+@step('no server bootstrap project exists')
+def recreate_boostrap_project_path(step):
+    bootstraps = world.bootstrap_projects_path
+    shutil.rmtree(bootstraps)
+    os.mkdir(bootstraps)
```

Comment:
This sandbox for bootstrap projects doesn't actually work yet, the suite as of now will use your `~/.config/ensime-vim`. Ideally it wouldn't, but this works for now.

`EnsimeLauncher` is parameterized already to allow a different path for the bootstrap project to be used, but it's hard to utilize that unless we were unit testing—can't easily pass a specific constructor param to a class at this level of testing. I think the sensible way to support this would be to allow setting the path with a `g:` variable, which we could set for tests. This could actually be useful for some users and other development/testing needs as well anyway. I could work on adding support for this on this branch or later.